Releases.md

```diff
@@ -19,6 +20,11 @@ See [DirectML version history on MSDN](https://docs.microsoft.com/windows/win32/
 | [1.1.0](#directml-110) | [DML_FEATURE_LEVEL_2_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_2_0) | Version 2004 (Build 10.0.19041; 20H1, "May 2020 Update") | -                                                                                         |
 | [1.0.0](#directml-100) | [DML_FEATURE_LEVEL_1_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_1_0) | Version 1903 (Build 10.0.18362; 19H1, "May 2019 Update") | -                                                                                         |
 
+# DirectML 1.9.1
+- UINT64 support for OneHot operator.
+- Improve FP32 convolution performance on NVIDIA graphics card.
+- Improve Concat performance on Intel DG2 and NVIDIA graphics cards.
```

Comment:
>Intel DG2 [](http://example.com/codeflow?start=32&length=9)

DG2 is an internal codename, which we shouldn't be using here. I don't think it's necessary to call out Intel and NVIDIA specifically, since this optimization isn't specific to them.